publish.sh

```diff
@@ -1,27 +1,18 @@
 #!/usr/bin/env bash
 
-# This script will publish to rubygems and dockerhub
-
-if [[ -z "${TAG_NAME:-}" ]]; then
-  echo "Please supply environment variable TAG_NAME."
-  echo "If you see this error in Jenkins it means the publish script was run"
-  echo "for a build that wasn't triggered by a tag - please check publish stage conditions."
-  exit 1
-fi
+set -e
 
-set -eux
+# This script will publish to rubygems and dockerhub
 
-DOCKERHUB_IMAGE="cyberark/parse-a-changelog"
+git clone git@github.com:conjurinc/release-tools.git
```

Comment:
should this clone the Atlantis repo now? https://github.cyberng.com/Conjur-Enterprise/release-tools

Reply:
No can do from outside github